src/operations/command.ts

```diff
@@ -97,9 +94,6 @@ export abstract class CommandOperation<T> extends AbstractOperation<T> {
     this.writeConcern = WriteConcern.fromOptions(options);
 
     // TODO(NODE-2056): make logger another "inheritable" property
```

Comment:
Actually we can remove this comment and close the associated JIRA ticket as "won't fix" or "gone away".